package.json

```diff
@@ -71,6 +71,7 @@
     "@vitejs/plugin-react": "^1.3.0",
     "jest": "^27.5.1",
     "jest-watch-typeahead": "^1.0.0",
+    "nodemon": "^2.0.15",
```

Comment:
@thanhsonng Can you update `package-lock.json` as well?

Reply:
Updated